client/tools/mgr-push/src/rhnpush/uploadLib.py

```diff
@@ -793,11 +798,14 @@ def packageCompare(pkg1, pkg2, is_mpm=None):
             e = str(e)
         evr = (e, str(pkg[1]), str(pkg[2]))
         packages.append(evr)
-    if is_mpm:
+    if pkgtype == "mpm":
         func = rhn_mpm.labelCompare
-    else:
-        from uyuni.common import rhn_rpm
+    elif pkgtype == "deb":
+        func = rhn_deb.label_compare
```

Comment:
IMO this should also be named `labelCompare`. Yes, that's not the style that's commonly used on Python, and it's also not what I would do, but being consistent is more important than sticking to snake_case.

After the traditional stack is removed, we will implement linting again and then we can define our style and update the code accordingly.

Reply:
Agreed. I changed it upon SonarCloud check failure but apparently it wasn't even retriggered.